bin/linja.in

```diff
@@ -463,7 +463,7 @@ def make_build_ninja(args):
 
         print("""rule LEAN""", file=f)
         print("""  depfile = ${DLEAN_FILE}""", file=f)
-        print("""  command = "%s" %s $in -o "${OLEAN_FILE}" """ \
+        print("""  command = "%s" %s -o ${OLEAN_FILE} $in""" \
```

Comment:
I'm actually using [file names with spaces](https://github.com/Kha/electrolysis/blob/master/ref/6%20Items%20And%20Attributes/6.1%20Items/6.1.2%20Modules/generated.lean) :) . The space could also be in the project root path, then you can still use imports.